cdhweb/pages/tests/test_templates.py

```diff
@@ -128,6 +128,40 @@ def test_upcoming_events(self, client, site, homepage, events):
         response = client.get(homepage.relative_url(site))
         assert events["deadline"] not in response.context["events"]
 
+    @pytest.mark.skip("fixme")
+    def test_empty_featured_pages(self, client, site, homepage):
```

Comment:
what's going wrong with the test? 

this approach seems reasonable enough; I wonder if there's a way to simplify and focus on just the methods/templates you worked on

Reply:
feel free to remove the skip and try it and see if you get the error I did — I didn't understand it at all

Reply:
I think the problem was the way you were removing the about page from the hierarchy and then trying to add it again after you deleted it. I'm not sure the right way to do that but since you're already testing that it handles both pages not being present, I simplified and used live/published instead of deleting.

Reply:
that makes sense! I'll do that then. thx for checking!